web/src/js/__tests__/ducks/ui/keyboardSpec.tsx

```diff
@@ -124,7 +124,7 @@ describe("onKeyDown", () => {
 
     it("should handle delete action", () => {
         store.dispatch(createKeyEvent("d"));
-        expect(fetchApi).toBeCalledWith("/flows/1", { method: "DELETE" });
+        expect(fetchApi).toBeCalledWith("flows/1", { method: "DELETE" });
```

Comment:
Calls to fetchApi should still be prefixed with `/`, fetchApi has explicit logic to make them relative.